jupyter-extension/package.json

```diff
@@ -40,13 +40,14 @@
     "clean:labextension": "rimraf jupyterlab_pachyderm/labextension",
     "clean:all": "npm run clean:lib && npm run clean:labextension",
     "cypress:ci": "cypress open",
-    "cypress:headless": "cypress run --browser chrome --headless --config video=true --record",
+    "cypress:headless": "cypress run --browser chrome --headless --record",
```

Comment:
Probably want to include another command for using electron here! Since chrome does not work in the arch linux vm

Reply:
is there something stopping us from just switching the browser here to electron?

Reply:
Hmm actually no and cypress is default always available choice for cypress given it runs headless without a GUI. Chrome and electron are pretty similar given that they are both Chromium based.